src/css/profile/mobile/common/timepicker.less

```diff
@@ -47,5 +47,35 @@
 	.ui-spin {
 		width: 100%;
 		height: 164 * @px_base;
+		justify-content: center;
+
+		.ui-time-picker-input {
+			width: 50 * @px_base;
+			height: 50 * @px_base;
+			align-self: center;
+			text-align: center;
+			font-size: 32 * @sp_base;
+			color: transparent;
+			.font(medium);
+			border-width: 0;
+			outline: unset;
+			outline-offset: unset;
+			text-shadow: 0 0 0 var(--primary-dark-color);
+			opacity: 0%;
```

Comment:
As I remember the opacity in percents makes issue on Tizen emulator. I have to check this.

Reply:
Confirmed. The opacity has to be in range 0 to 1 without percent

Reply:
Do we have to change the opacity value to be in range 0 to 1?
Okay I will change it.